### ISSUE TITLE
Respect boundary-space declaration in output pane

### ISSUE THREAD

**Issue by joewiz** [eXist-db/eXide#134]
eXide's current pretty printing of XML nodes nullifies boundary space declarations, making it ill suited to teaching this concept.  Compare the results of the following query:

``` xquery
xquery version "3.1";

declare boundary-space preserve;

<opener><salute>Dear</salute> <title>Mr.</title> <persName reg="Obama, Barack" role="president">Obama</persName>,</opener>
```

**eXide**

> <img width="569" alt="screen shot 2016-09-10 at 2 47 11 pm" src="https://cloud.githubusercontent.com/assets/59118/18412781/93071d4c-7765-11e6-890d-2e98d0d51d8c.png">

**View source (Chrome)**

> <img width="949" alt="screen shot 2016-09-10 at 2 47 26 pm" src="https://cloud.githubusercontent.com/assets/59118/18412777/8e8ea6b8-7765-11e6-842c-f66135d495b4.png">

I propose a non-indented option that can be enabled either by default or when boundary space is critical to the results of a query.

**Comment by joewiz** [comment]
Would be fixed by #147.

**Comment by joewiz** [comment]
Closed by https://github.com/wolfgangmm/eXide/pull/168.

**Comment by joewiz** [comment]
Closed by https://github.com/wolfgangmm/eXide/pull/168.

**Comment by joewiz** [comment]
Closed by #168.
